delfin/drivers/fake_storage/__init__.py

```diff
@@ -422,17 +422,17 @@ def list_shares(self, ctx):
         return share_list
 
     def add_trap_config(self, context, trap_config):
-        pass
+        pass # Fakedriver do not require to add trap config
```

Comment:
CI is failed (E261 at least two spaces before inline comment), please fix.

Reply:
Updated spaces before inline comments. Thanks for guiding!